include/raft.h

```diff
@@ -843,7 +834,7 @@ int raft_node_has_sufficient_logs(raft_node_t* me_);
  * @return
  *  0 on success;
  *  RAFT_ERR_SHUTDOWN when server MUST shutdown */
-int raft_apply_all(raft_server_t* me_);
+int raft_apply(raft_server_t* me_);
```

Comment:
I renamed it so we don't really differentiate between applying one or all entries. 'apply' basically means it applies all entries up to the commit index. This could mean that a node has been added/removed several times (but not actually removed from the node list just yet). We track membership changes with each log entry and simply apply the "final" state to the state machine which is the effective state after all log entries have been applied.